backend/src/existing.rs

```diff
@@ -71,6 +71,7 @@ impl Highway {
 }
 
 /// This determines what's in the graph. The cost function is just based on distance.
+// TODO Incorporate gradient
```

Comment:
_all_ of the routing should be calculating a speed to cross each edge, and mostly using that. I still need to go pull in the math from Rosa that does this.